multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ModuleHooksAggregator.java

```diff
@@ -113,30 +105,4 @@ private List<HookPhase> getExecutedHookPhasesForHook(Map<String, List<String>> a
         return mapToHookPhases(executedHookPhasesForHook);
     }
 
-    private void updateExecutedHooksForModule(Map<String, List<String>> alreadyExecutedHooks, List<HookPhase> currentHookPhasesForExecution,
-                                              List<Hook> hooksForExecution) {
-        Map<String, List<String>> result = new HashMap<>(alreadyExecutedHooks);
-        updateExecutedHooks(result, currentHookPhasesForExecution, hooksForExecution);
-        StepsUtil.setExecutedHooksForModule(execution, moduleToDeploy.getName(), result);
-    }
-
-    private void updateExecutedHooks(Map<String, List<String>> alreadyExecutedHooks, List<HookPhase> currentHookPhasesForExecution,
-                                     List<Hook> hooksForExecution) {
-        for (Hook hook : hooksForExecution) {
-            updateHook(alreadyExecutedHooks, currentHookPhasesForExecution, hook);
-        }
-    }
-
-    private void updateHook(Map<String, List<String>> alreadyExecutedHooks, List<HookPhase> currentHookPhasesForExecution, Hook hook) {
-        List<String> hookPhasesBasedOnCurrentHookPhase = getHookPhasesBasedOnCurrentHookPhase(currentHookPhasesForExecution,
-                                                                                              hook.getPhases());
-        alreadyExecutedHooks.merge(hook.getName(), hookPhasesBasedOnCurrentHookPhase, ListUtils::union);
-    }
-
-    private List<String> getHookPhasesBasedOnCurrentHookPhase(List<HookPhase> currentHookPhasesForExecution, List<String> hookPhases) {
-        return hookPhases.stream()
-                         .filter(phase -> currentHookPhasesForExecution.contains(HookPhase.fromString(phase)))
-                         .collect(Collectors.toList());
-    }
-
```

Comment:
These methods are moved to HooksExecutor.  Is it because it makes more structural sense to group them there?